src/consts.js

```diff
@@ -211,6 +211,9 @@ export const ACTOR_LIMITS = {
     // For each build or run container, set disk quota based on memory size
     RUN_DISK_TO_MEMORY_SIZE_COEFF: 2,
 
+    // For each build or run container, set CPU cores based on memory size
+    RUN_MEMORY_SIZE_TO_CPU_CORES_COEFF: 4096,
```

Comment:
Just one note - we always thrive to make constants contain units, otherwise, Mars probes can crash (https://en.wikipedia.org/wiki/Mars_Climate_Orbiter#Cause_of_failure).

How about calling this `RUN_MEMORY_MBYTES_PER_CPU_CORE` ?

Reply:
That makes sense, done in 645a128.